### PR TITLE
Add Rart75 hotswap and support via

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

Add New keyboard rart75 Hotswap version with atmega32a, support layout via

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/rart/rart75hs/config.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#define ENCODERS_PAD_A { C1 }
+#define ENCODERS_PAD_B { C2 }
+#define ENCODER_RESOLUTION 4 //default/suggested
+
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 1
+
+#define RGB_DI_PIN C0
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/rart/rart75hs/config.h:62]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/rart/rart75hs/readme.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# [RART 75 Hotswap](https://github.com/alabahuy/RART/tree/master/RART75)
+
+![RART 75](https://user-images.githubusercontent.com/30220306/157791657-8c7f9093-d42f-41f1-8252-81fbb21cb080.jpg)
```
Comment (drashna) [inline keyboards/rart/rart75hs/readme.md:3]: Please rehost this on an image hosting site, such as Imgur.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/rart/rart75hs/info.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                {"label" : "Ctrl", "x":0, "y":5.25, "w":1.25},
+                {"label" : "Windows", "x":1.25, "y":5.25, "w":1.25},
+                {"label" : "Alt", "x":2.5, "y":5.25, "w":1.25},
+                {"label" : "Space", "x":3.75, "y":5.25, "w":6.25},
+                {"label" : "Alt", "x":10, "y":5.25 },
+                {"label" : "Windows","x":11, "y":5.25 },
+                {"label" : "Ctrl", "x":12, "y":5.25 },
+                {"label" : "Left", "x":13.25, "y":5.5 },
+                {"label" : "Down", "x":14.25, "y":5.5 },
+                {"label" : "Right", "x":15.25, "y":5.5 }
```
Comment (fauxpark) [inline keyboards/rart/rart75hs/info.json:95]:
```suggestion
                {"label" : "Right", "x":15.25, "y":5.5 }
```

File: keyboards/rart/rart75hs/info.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                {"label" : "F5", "x":5.5, "y":0},
+                {"label" : "F6", "x":6.5, "y":0},
+                {"label" : "F7", "x":7.5, "y":0},
+                {"label" : "F8", "x":8.5, "y":0},
+                {"label" : "F9", "x":9.75, "y":0},
+                {"label" : "F10", "x":10.75, "y":0},
+                {"label" : "F11", "x":11.75, "y":0},
+                {"label" : "F12", "x":12.75, "y":0},
+                {"label" : "Delete", "x":14, "y":0},
+                {"label" : "Encoder", "x":15.5, "y":0},
```
Comment (fauxpark) [inline keyboards/rart/rart75hs/info.json:22]:
```suggestion
                {"label" : "Encoder", "x":15.5, "y":0},
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/rart/rart75hs/info.json (hunk 4 above)
Comment (alabahuy) [reply keyboards/rart/rart75hs/info.json]: done :) @fauxpark

File: keyboards/rart/rart75hs/readme.md (hunk 2 above)
Comment (alabahuy) [reply keyboards/rart/rart75hs/readme.md]: done :) @drashna

**Review by drashna — APPROVED** [review]

**Comment by alabahuy** [comment]
any update @fauxpark ?

File: keyboards/rart/rart75hs/readme.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# [RART 75 Hotswap](https://github.com/alabahuy/RART/tree/master/RART75)
+
+![RART 75](https://imgur.com/FVWlm8U)
```
Comment (fauxpark) [inline keyboards/rart/rart75hs/readme.md:3]:
```suggestion
![RART 75](https://i.imgur.com/FVWlm8Uh.jpeg)
```

File: keyboards/rart/rart75hs/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+  [3] = LAYOUT_all(
+    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,  _______,  _______,  _______,  _______,  _______,
+    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,  _______,
+    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,
+    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,                      _______,
+    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,                      _______,  _______,
+    _______,  _______,  _______,            _______,                                _______,  _______,  _______,  _______,                      _______,  _______
+  )
+
```
Comment (fauxpark) [inline keyboards/rart/rart75hs/keymaps/default/keymap.c:54]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/rart/rart75hs/keymaps/default/keymap.c (hunk 6 above)
Comment (alabahuy) [reply keyboards/rart/rart75hs/keymaps/default/keymap.c]: Done :) @fauxpark

**Review by zvecr — APPROVED** [review]

**Pull request merged**